docs/development/developer-guide.rst

```diff
@@ -131,3 +131,35 @@ simple Python script ``tools/vendor.py``.
 To refresh the dependencies, run the following command::
 
     $ tox -e vendor
+
+----------------
+Type annotations
+----------------
+
+Most standards and best practices are enforced by
+[Ruff](https://docs.astral.sh/ruff/rules/)'s ``ANN2``, ``FA``, ``PYI``, ``UP``
```

Comment:
I think this is markdown syntax for link, but setuptools uses RST.
There is a couple of other occurrences below.

Reply:
Not used to RST, but I think I got it right

Reply:
Thank you.

Reminder for my future self, probably adding something like https://github.com/sloria/sphinx-issues is useful.